cmd/boost/provider_cmd.go

```diff
@@ -97,3 +109,192 @@ var libp2pInfoCmd = &cli.Command{
 		return nil
 	},
 }
+
+var queryAskCmd = &cli.Command{
+	Name:      "query-ask",
+	Usage:     "Find a miners ask",
+	ArgsUsage: "[minerAddress]",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "peerid",
+			Usage: "specify peer ID of node to make query against",
+		},
```

Comment:
It seems like the peerid flag is ignored in the code, and we don't have a peerid flag for retrieval ask, so I suggest we just delete it